app/assets/javascripts/users.js

```diff
@@ -0,0 +1,25 @@
+function getRepositories() {
+  var url = $("#repositories").data("repositories")
+  $.getJSON(url, function(data) {
+    $.each(data, function(_idx, repository) {
```

Comment:
It's cool but a cooler way is to add a partial and render that partial with `remote: :true` and just call the element like:
```
$("#repositories").append("<%= j render cooler_repo(repositorie)  %>");
```